.github/workflows/lint.yml

```diff
@@ -9,13 +9,9 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
-      - name: Set up Python 3.11
+      - name: Set up Python 3.8
```

Comment:
All our linters are set up for 3.8, so did this for consistency.
However, it seems like a good practice to set it to the latest python version (https://black.readthedocs.io/en/stable/integrations/source_version_control.html), this might be tackled in further initiatives

Reply:
Yes I was going to say it 👍